pkg/volume/util/util.go

```diff
@@ -945,6 +945,13 @@ func CheckPersistentVolumeClaimModeBlock(pvc *v1.PersistentVolumeClaim) bool {
 	return utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) && pvc.Spec.VolumeMode != nil && *pvc.Spec.VolumeMode == v1.PersistentVolumeBlock
 }
 
+func IsWindowsNamedPipe(goos, path string) bool {
+	if goos == "windows" && strings.HasPrefix(path, `\\.\pipe\`) {
```

Comment:
@PatrickLang @atomaras  Does named pipe also starts with `\\.\pipe\`? what about upper case like `\\.\Pipe\` ?

Reply:
It also supports like this: `\.\pipe\` ...
```
docker run -it --rm -v \.\pipe\docker_engine:\.\pipe\docker_engine microsoft:windowsservercore:1803 powershell
```

Reply:
The underlying `CreateFileW()` is case insensitive by default
https://docs.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-createfilew